pkg/kubernetes/kubernetes.go

```diff
@@ -21,7 +21,7 @@ import (
 )
 
 const (
-	daprLatestVersion = "latest"
+	daprLatestVersion = "vlatest"
```

Comment:
what is "vlatest" ?, is this right ?

Reply:
yes, in current CLI, if a user runs `dapr init -k`, cli tries to get the 'vlatest' version of dapr. This change makes it so that the user can run `dapr init -k` without supplying a runtime version.

Reply:
looks like it is changed. please take latest changes and update the PR and then this is not needed.